src/main/java/com/dumbdogdiner/stickyapi/common/command/tabcomplete/TabExecutor.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.command.tabcomplete;
+
+import com.dumbdogdiner.stickyapi.common.arguments.Arguments;
+import org.bukkit.command.CommandSender;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+@FunctionalInterface
+public interface TabExecutor<T> {
```

Comment:
```suggestion
public interface TabExecutor<Sender> {
```
Is fine to keep as `T`, as long as Executor uses the same parameter name.

Reply:
I was trying to make types more specific I think, I can't see the full file, I'm mobile